hack/verify-protobuf.sh

```
@@ -2,29 +2,10 @@
-echo "Checking against freshly generated protobuf..."
-for gv in ${API_GROUP_VERSION_PATHS}; do
-  ret=0
-  diff -Naupr -I 'Auto generated by' -x 'zz_generated.*' -x 'types_swagger*' "${SCRIPT_ROOT}/${gv}" "${TMP_ROOT}/${gv}" || ret=$?
-  if [[ $ret -ne 0 ]]; then
-    cp -a "${TMP_ROOT}"/* "${SCRIPT_ROOT}/"
-    echo "Protobuf is out of date. Please run hack/update-protobuf.sh"
-    exit 1
-  fi
-done
-echo "Protobuf up to date."
+GENERATOR=go-to-protobuf EXTRA_ARGS=--verify ${SCRIPT_ROOT}/hack/update-codegen.sh
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)